--- spack-src/cpp/cmake_modules/BuildUtils.cmake	2025-10-20 07:48:39
+++ /Users/gitlab-runner-p-0/BuildUtils.cmake	2026-04-07 10:19:32
@@ -110,7 +110,7 @@
     execute_process(COMMAND ${LIBTOOL_MACOS} -V
                     OUTPUT_VARIABLE LIBTOOL_V_OUTPUT
                     OUTPUT_STRIP_TRAILING_WHITESPACE)
-    if(NOT "${LIBTOOL_V_OUTPUT}" MATCHES ".*cctools-([0-9.]+).*")
+    if(NOT "${LIBTOOL_V_OUTPUT}" MATCHES ".*cctools-([0-9.]+).*" AND NOT "${LIBTOOL_V_OUTPUT}" MATCHES ".*cctools_ld-([0-9.]+).*")
       message(FATAL_ERROR "libtool found appears not to be Apple's libtool: ${LIBTOOL_MACOS}"
       )
     endif()
